htdocs/supplier_proposal/class/supplier_proposal.class.php

```diff
@@ -2260,7 +2260,7 @@ function load_board($user,$mode)
                 $response->nbtodo++;
                 if ($mode == 'opened')
                 {
-                    $datelimit = $this->db->jdate($obj->datefin);
+                    $datelimit = $this->db->jdate($obj->datec);
```

Comment:
Why not using  $datelimit = $this->db->jdate($obj->date_valid);  ?
This field seems to exists in database.